OWML.Launcher/App.cs

```diff
@@ -70,6 +72,64 @@ private void LocateGamePath()
             }
         }
 
+        private void CheckGameVersion()
+        {
+            var versionReader = new GameVersionReader(_writer, new BinaryPatcher(_owmlConfig, _writer));
+            var gameVersion = versionReader.GetGameVersion();
+            _writer.WriteLine($"Game version: {gameVersion}", MessageType.Info);
+            var splitGameVersion = gameVersion.Split('.');
+            var splitMinVersion = _owmlManifest.MinimalGameVersion.Split('.');
+            var splitMaxVersion = _owmlManifest.MaximalGameVersion.Split('.');
+            var count = Math.Min(splitGameVersion.Length, Math.Min(splitMaxVersion.Length, splitMinVersion.Length));
+            splitGameVersion[count - 1] = string.Join("", splitGameVersion.Skip(count - 1));
+            splitMinVersion[count - 1] = string.Join("", splitMinVersion.Skip(count - 1));
+            splitMaxVersion[count - 1] = string.Join("", splitMaxVersion.Skip(count - 1));
+            for (var i = 0; i < count; i++)
+            {
+                var gameNumber = Convert.ToInt32(splitGameVersion[i]);
+                var minNumber = Convert.ToInt32(splitMinVersion[i]);
+                var maxNumber = Convert.ToInt32(splitMaxVersion[i]);
+                if (gameNumber < minNumber)
+                {
+                    ThrowBelowError();
+                }
+                if (gameNumber > maxNumber)
+                {
+                    ThrowAboveWarning(gameVersion);
+                    return;
+                }
+            }
+        }
+
+        private void ThrowBelowError()
+        {
+            _writer.WriteLine("Unsupported game version found", MessageType.Error);
+            _writer.WriteLine("Press any key to exit...", MessageType.Info);
+            Console.ReadKey();
+            ExitConsole();
+        }
+
+        private void ThrowAboveWarning(string currentVersion)
+        {
+            _writer.WriteLine("Potentially unsupported game version found", MessageType.Warning);
+            ConsoleKey response;
+            do
+            {
+                _writer.WriteLine("Continue loading? [y/n]", MessageType.Info);
+                response = Console.ReadKey(false).Key;
+                if (response != ConsoleKey.Enter)
+                {
+                    Console.WriteLine();
+                }
+            } while (response != ConsoleKey.Y && response != ConsoleKey.N);
+            if (response == ConsoleKey.N)
+            {
+                ExitConsole();
+            }
+            _owmlManifest.MaximalGameVersion = currentVersion;
+            JsonHelper.SaveJsonObject(Constants.OwmlManifestFileName, _owmlManifest);
```

Comment:
Why change the manifest here? We don't trust users to know whats supported or not?

Reply:
Agreed, I don't think we should ever edit manifest files.

Reply:
If something like this is needed, some other file should be used to persist the data (like the config file or something else)

Reply:
I guess it's to only prompt the user once. 
I think we should not bother the user at all, see above comment.

Reply:
I will just turn it into warning then